pkgs/servers/kanidm/default.nix

```diff
@@ -17,24 +17,28 @@ let
 in
 rustPlatform.buildRustPackage rec {
   pname = "kanidm";
-  version = "1.1.0-alpha.11";
+  version = "1.1.0-alpha.12";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = "refs/tags/v${version}";
-    hash = "sha256-TVGLL1Ir/Nld0kdhWmcYYmChrW42ctJPY/U7wtuEwCo=";
+    hash = "sha256-ZlUn7m5xgMWWIr9y/dkM/yZ2KF2LdkaxqtHsMcxAT/M=";
   };
 
   cargoLock = {
     lockFile = ./Cargo.lock;
     outputHashes = {
-      "tracing-forest-0.1.4" = "sha256-ofBLxSzZ5SYy8cbViVUa6VXKbOgd8lt7QUYhL0BW6I4=";
+      "tracing-forest-0.1.5" = "sha256-L6auSKB4DCnZBZpx7spiikhSOD6i1W3erc3zjn+26Ao=";
     };
   };
 
   KANIDM_BUILD_PROFILE = "release_nixos_${arch}";
 
+  patches = [
+    ./cpu-opt-level.patch
```

Comment:
Could you include this via fetchpatch (e.g. with `https://github.com/bjornfor/kanidm/commit/5d7ecc83d69bcc23da439abb8674a6d7da0951f6.patch`)? This is more common for patches that we try to bring upstream.

I can't really say wether this change is desirable at all, according to this [discussion](https://github.com/kanidm/kanidm/pull/1590#issuecomment-1529576062).

Reply:
Maybe we should drop the patch and use CPU optimization `none` instead?

Reply:
Yeah, that sounds easier to me tbh. If anybody cares a lot about performance, they could always reintroduce a solution again.

Reply:
This is not resolved yet. Either you'd need to `fetchpatch` https://github.com/kanidm/kanidm/commit/59c6723f7dfb2266eae45c3b2ddd377872a7a113 or change our build profile to  `none`.

Reply:
Went fetchpatching the referenced commit, given that this is what went upstream.